### ISSUE TITLE
Issues in training and loss/accuracy computation

### ISSUE THREAD

**Issue by Ektagavas** [wannabeOG/MAS-PyTorch#4]
In the file `model_train.py`, in the training loop, I see below code in L175-179:

`output = model.tmodel(input_data)`
`_, preds = torch.max(output, 1)`
`running_corrects += torch.sum(preds == labels.data)`
`loss = model_criterion(output, labels)`

What I understood is model.tmodel is the feature extractor, but yet we are comparing the outputs with target labels. Shouldn't we compare it with the last layer (classifier) output?

**Comment by Ektagavas** [comment]
I think I got now. `tmodel`is not feature extractor actually but a full network. 
Apologies for the confusion. :(
